src/common/threading.jl

```diff
@@ -0,0 +1,10 @@
+import FFTW
+using LinearAlgebra
+
+function setup_threading(;n_fft=1, n_blas=Threads.nthreads())
+    n_julia = Threads.nthreads()
+    FFTW.set_num_threads(n_fft)
+    BLAS.set_num_threads(n_blas)
+    mpi_master() && @info "Threading setup" n_fft n_blas n_julia
```

Comment:
That's a bit barebones. Either print what these numbers are ("n_fft=$n_fft, n_blas=$n_blas, n_julia=$n_julia") or don't write anything

Reply:
But this is exacty what happens

Reply:
Oh I didn't know that, cool!